### ISSUE TITLE
Expose values via Modbus TCP

### ISSUE THREAD

**Issue by ck-coke** [networkupstools/nut#1857]
Hi,

is it possible to expose UPS values either via NUT or mbusd as Modbus TCP server?
In parallel to phoenixcontact_modbus - Driver for Phoenix Contact.

BR

**Comment by jimklimov** [comment]
I don't think this was pursued in NUT core, but can be done as a separate client looping like upsmon to ask for data and publishing a different protocol, or even pass back commands (if NUT auth is taken care of). I believe somebody hacked MQTT scripts as a PoC. There are several bindings (C, C++, Python, Perl in-tree, jNut now separately, etc.) of varied completeness for NUT protocol you might use. I guess a C client could be welcome as a PR to maintain in-core.
